hack/update-deps.sh

```diff
@@ -31,8 +31,8 @@ cd "${ROOT_DIR}"
 # The list of dependencies that we track at HEAD and periodically
 # float forward in this repository.
 FLOATING_DEPS=(
-  "knative.dev/test-infra@release-0.17"
-  "knative.dev/pkg@release-0.17"
+  "knative.dev/test-infra@master"
```

Comment:
FWIW, my preference is:

```suggestion
  "knative.dev/test-infra@${VERSION}"
```